src/lib/isKeywordColor.js

```diff
@@ -0,0 +1,160 @@
+import assertString from './util/assertString';
+
+// https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#color_keywords
+const keywords = [
+  'transparent',
+  'currentcolor',
```

Comment:
I'm not really a big fan of stacking checking for sure...

Reply:
Yeah, not my nicest pice of code 😅 
But I can't find a better way to do this :-)

Reply:
We should probably replace the array with a set. set.has is more efficient than indexOf

Reply:
Good suggestion, I've just replaced it :-)